regions/shapes/annulus.py

```diff
@@ -0,0 +1,89 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import math
+import operator
+
+import numpy as np
+
+from astropy import wcs
+from astropy import coordinates
+from astropy import units as u
+
+from ..core import CompoundPixelRegion, CompoundSkyRegion
+from ..shapes import CirclePixelRegion, CircleSkyRegion
+
+
+class AnnulusPixelRegion(CompoundPixelRegion):
+    """
+    An annulus in pixel coordinates.
+
+    Parameters
+    ----------
+    center : :class:`~regions.core.pixcoord.PixCoord`
+        The position of the center of the annulus.
+    inner_radius : float
+        The inner radius of the annulus
+    outer_radius : float
+        The outer radius of the annulus
+    """
+
+    def __init__(self, center, inner_radius, outer_radius, meta=None, visual=None):
```

Comment:
I don't really understand the use case for this, what does ``contains`` e.g. imply for 5 annuli. Couldn't we return a DS9 n-annulus as a list of n in/out annuli?